models/turbine_models/custom_models/sd_inference/sd_pipeline.py

```diff
@@ -190,6 +195,7 @@ def get_sd_model_map(hf_model_name):
         "stabilityai/sdxl-turbo",
         "stabilityai/stable-diffusion-xl-base-1.0",
         "/models/SDXL/official_pytorch/fp16/stable_diffusion_fp16/checkpoint_pipe",
+        "/models/SDXL/official_pytorch/fp16/stable_diffusion_fp16//checkpoint_pipe",
```

Comment:
We should try not matching hardcoded path's. Can we add a flag for the user to set which sd version to use for the model map so that downloaded model cards can be used with any path name?